book/_config.yml

```diff
@@ -15,9 +15,9 @@ parse:
     github_org_url: "https://github.com/CryoInTheCloud"
     book_repo: "CryoCloudWebsite"
     website_url: "https://cryointhecloud.github.io"
-    jupyterhub_url: "https://cryo.2i2c.cloud"
+    jupyterhub_url: "https://cryonthecloud.2i2c.cloud"
     slack_workspace_url: "https://CryoCloud.slack.com"
-    docker_image: "https://hub.docker.com/r/cryocloud"
+    docker_image: "https://hub.docker.com/r/cryointhecloud"
```

Comment:
Did you update the `DOCKER_USERNAME` and `DOCKER_PASSWORD` secrets for this new Docker Hub user account? Or are they under the same account?

Reply:
The config wasn't correct and I don't think that line is used for anything in Github is it? The full link is: https://hub.docker.com/repository/docker/cryointhecloud/cryocloudwebsite. I think the config file is solely for the book and is filling in variables in certain md files. Our DOCKER_IMAGE secret is set to cryointhecloud/cryointhecloud and hasn't been changed.

Reply:
Ah ok, I thought we were pushing to `cryocloud` before, but I see the docker image is at https://hub.docker.com/r/cryointhecloud/cryocloudwebsite/tags. Sorry for the confusion!